integration_tests/e2e/goal/createGoals.cy.ts

```diff
@@ -61,7 +61,7 @@ context('Create goals', () => {
       .hasFieldInError('goals[0].steps[0].title')
   })
 
-  it('should not be able to add an empty step to a goal given form has validation errors', () => {
+  it('should be able to add an empty step, and validation only performed on final form submission', () => {
```

Comment:
This cypress test was fundamentally wrong! 🤣 , and so I've changed it to reflect the fact that you can add steps etc with the form in any state, and it is not until you submit the entire form that validation happens 👍